src/native/tabs.rs

```diff
@@ -195,23 +197,23 @@ where
     /// Sets the font of the icons of the
     /// [`TabLabel`](super::tab_bar::TabLabel)s of the
     /// [`TabBar`](super::tab_bar::TabBar).
-    pub fn icon_font(mut self, icon_font: Font) -> Self {
+    pub fn icon_font(mut self, icon_font: Renderer::Font) -> Self {
         self.tab_bar = self.tab_bar.icon_font(icon_font);
         self
     }
 
     /// Sets the font of the text of the
     /// [`TabLabel`](super::tab_bar::TabLabel)s of the
     /// [`TabBar`](super::tab_bar::TabBar).
-    pub fn text_font(mut self, text_font: Font) -> Self {
+    pub fn text_font(mut self, text_font: Renderer::Font) -> Self {
         self.tab_bar = self.tab_bar.text_font(text_font);
         self
     }
 
     /// Sets the style of the [`TabBar`](super::tab_bar::TabBar).
     pub fn tab_bar_style<T>(mut self, style: T) -> Self
     where
-        T: Into<<Renderer as crate::native::tab_bar::Renderer>::Style>,
+        T: Into<Style>,
```

Comment:
`Into<Style>` is wrong here. `Style` is the specific style of a `StyleSheet` e.g. when the element is hovered or active. You have to use the trait `StyleSheet` instead:
```Rust
    style_sheet: impl Into<Box<dyn StyleSheet + 'a>>
```

See the card implementation as a reference.